# lets use the flags the EB sets for us
# By Ward Poelmans <wpoely86@gmail.com>
diff -ur c37b2.orig/build/UNX/Makefile_em64t c37b2/build/UNX/Makefile_em64t
--- c37b2.orig/build/UNX/Makefile_em64t	2012-07-10 14:06:18.000000000 +0200
+++ c37b2/build/UNX/Makefile_em64t	2013-12-03 15:35:18.997245768 +0100
@@ -39,8 +39,8 @@
 
    FC0 = $(FC) -c -O0 -free
    FC1 = $(FC) -c -O1 -free
-   FC2 = $(FC) -c -O3 -mp1 -axSSE4.1 -free 
-   FC3 = $(FC) -c -O3 -mp1 -axSSE4.1 -free
+   FC2 = $(FC) -c -O3 -mp1 -free $(F90FLAGS)
+   FC3 = $(FC) -c -O3 -mp1 -free $(F90FLAGS)
    FCR = $(FC) -c -u -V -free
    FCD = $(FC) -c -g -O0 -u -traceback -free
 ifdef DEBUG
